motr/st/cancel.c

```diff
@@ -164,7 +164,7 @@ static void m0_write_cancel(void)
 		st_entity_open(&objs[i].ob_entity);
 		/* Create the write request */
 		st_obj_op(&objs[i], M0_OC_WRITE, &ext, &data,
-			  NULL, 0, 0, &ops[i]);
+			  &attr, 0, 0, &ops[i]);
```

Comment:
It seems this attr has size is 1, it should be size of md5 checksum.
Line 144: rc = m0_bufvec_alloc(&attr, blk_cnt, 1);

Reply:
done